source/recall.js

```diff
@@ -0,0 +1,35 @@
+import uncurryN from './uncurryN';
+
+/**
+ * Collapses a high-order unary function. The order N of the function
+ * is implicit. A maximum number of N values are singularly fed.
+ *
+ * @func
+ * @category Function
+ * @sig ((a -> b -> * -> c), *...) -> d
+ * @param {Function} fn The N-order function to be collapsed.
+ * @param {...*} args Up to N input values.
+ * @return {*}
+ * @see R.call
+ * @example
+ *
+ *      const xs = [{a: 1}, {a: 2}, {a: 3}];
+ *      const f = R.compose(R.find, R.propEq('a'));
+ *      R.recall(f, 2, xs); //=> {"a": 2}
+ *
+ * @symb R.recall(f, a, b) = f(a)(b)
+ */
+var recall = uncurryN(2, function(fn) {
+  return function() {
+    var numArgs = arguments.length;
```

Comment:
I'm quite sure that we need to supply the arity here as we do with `uncurryN`.

What happens if our target output is a function?  What if we want to convert

```js
const makeHandler = (a) => (b) => (evt) => ({do: 'something', with: 'a, b, and evt'})
```

so that it can be used as 

```js
something .addHandler (recall (makeHandler, 42, false) )
```

where `addHandler` accepts a function?

Reply:
This is a good use. The value returned can be whatever it needs to be, including just another function. If we supply a function `a => b => c => d => a + b * c + d` and want a function a particular depth, we just provide values up to that depth. For example, `[ 2, 3 ]` if we want the function `c => d => 5 * c + d`.

Reply:
I think I misread before and didn't notice `&& i < numArgs`.  My initial reading suggested that this use case wouldn't be possible with that implementation.